apis/appmesh/v1beta2/virtualgateway_types.go

```diff
@@ -58,41 +65,31 @@ type VirtualGatewayCondition struct {
 // VirtualGatewayHealthCheckPolicy refers to https://docs.aws.amazon.com/app-mesh/latest/userguide/virtual_gateways.html
 type VirtualGatewayHealthCheckPolicy struct {
 	// The number of consecutive successful health checks that must occur before declaring listener healthy.
-	// If unspecified, defaults to be 10
 	// +kubebuilder:validation:Minimum=2
 	// +kubebuilder:validation:Maximum=10
 	// +optional
-	HealthyThreshold *int64 `json:"healthyThreshold,omitempty"`
+	HealthyThreshold int64 `json:"healthyThreshold"`
 	// The time period in milliseconds between each health check execution.
-	// If unspecified, defaults to be 30000
```

Comment:
why are we removing the default values?

Reply:
These changes have been borrowed from https://github.com/aws/aws-app-mesh-controller-for-k8s/pull/215.